agent/metasploit_agent.py

```diff
@@ -122,7 +129,7 @@ def process(self, message: m.Message) -> None:
                 ):
                     technical_detail = f"Using `{module_instance.moduletype}` module `{module_instance.modulename}`\n"
                     technical_detail += f"Target: {vhost}:{rport}\n"
-                    technical_detail += f'Message: \n```{results["message"]}```'
+                    technical_detail += f'Message: \n```\n{results["message"]}\n```'
```

Comment:
```suggestion
                    technical_detail += f'Message: \n```shell\n{results["message"]}\n```'
```